api/swagger-spec/v1.json

```diff
@@ -5707,6 +5707,54 @@
       "nickname": "connectGetNamespacedPodExec",
       "parameters": [
        {
+        "type": "boolean",
+        "paramType": "query",
+        "name": "stdin",
+        "description": "redirect the standard input stream of the pod for this call; defaults to false",
```

Comment:
I believe it means that stdin is redirected into the process you are executing in the pod.
However, this PR is not introducing a new description for this (maybe should be clarified in a separate PR). This PR simply exposes these parameters and descriptions that were already in the type annotations. 
`api/swagger-spec/v1.json` is a generated file.